chatcommands.py

```diff
@@ -1162,6 +1162,18 @@ def checkpost(msg, url, alias_used='scan'):  # FIXME: Currently does not support
     url = to_protocol_relative(post_data.post_url)
     post = Post(api_response=post_data.as_dict)
 
+    if has_already_been_posted(post_data.site, post_data.post_id, post_data.title) \
+            and not is_false_positive((post_data.post_id, post_data.site)):
+        # Don't re-report if the post wasn't marked as a false positive. If it was marked as a false positive,
+        # this force scan might be attempting to correct that/fix a mistake/etc.
+
+        response_text = "This post is already recently reported"
```

Comment:
Since this is the `!!/scan` command rather than the `!!/report` command, this should probably be 'This post has been recently scanned'

Reply:
I noticed this and deliberately kept the word *reported*, as it really *is* reported. Scanning a post won't prevent you from scanning again, but a report does. So I'd like to not change it.